### DISCUSSION TITLE
Enhancement: Switch Assistants Mid Conversation

### DISCUSSION THREAD

**Discussion by airwindk** [danny-avila/LibreChat#4429] [Feature Requests & Suggestions]
### What features would you like to see added?

Ref #4422 
 
Similar to the ability to toggle between endpoints mid-conversation, add a new chat setting that allows you to toggle across assistants mid-conversation 

### More details

N/a

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
You can do this already by switching via dropdown/mention command.

![image](https://github.com/user-attachments/assets/6cf5bedf-f031-46c0-a958-70115efe0506)

From the discussion:

> At this point I would expect to be redirected to Assistant 2's default page

I would not expect this and would just start a new chat with the desired assistant.
